src/styles/utils/_variables.scss

```
@@ -0,0 +1,32 @@
+// stars
+$star-size: 16px;
+$star-indent: $star-size / 4;
+
+//header
+$header-padding: 0 50px;
+
+$image-size: 40px;
+$image-margin: $image-size * 0.25;
+
+$nav-link--height: 60px;
+$nav-item--indent: $nav-link--height / 3;
```

use only one `-` in naming
```suggestion
$nav-link-height: 60px;
$nav-item-indent: $nav-link--height / 3;
```